### ISSUE TITLE
pretrained model for wider pedestrain

### ISSUE THREAD

**Issue by annie7777** [hasanirtiza/Pedestron#14]
Hi thanks for your nice work. Are you planning to add pretrained model purely trained on wider pedestrain? Also what is the difference between CrowdHuman 1 and CrowdHuman 2 in the pretrained-model?

**Comment by hasanirtiza** [comment]
Yes, we are planning to release the pre-trained model for WiderPedestrian as well. Stay notified or keep watching the repository. As for CrowdHuman1 and CrowdHuman 2, both are models trained at different point in time. So you might observe a slight performance difference between two models.

**Comment by hasanirtiza** [comment]
Pre-trained model for wider pedestrian has been released.

**Comment by annie7777** [comment]
> Pre-trained model for wider pedestrian has been released.

Thanks very much. Just tried it,  it works the best on my own dataset. But people in my our image is very small like 10*5 pixels, is there a way to increase the input image size so that the network can detect small persons in the test image?

**Comment by hasanirtiza** [comment]
>  is there a way to increase the input image size so that the network can detect small persons in the test image?

In the config file you can change the resolution of the image. By the way which dataset are you testing on ?

**Comment by annie7777** [comment]
> > is there a way to increase the input image size so that the network can detect small persons in the test image?
> 
> In the config file you can change the resolution of the image. By the way which dataset are you testing on ?

Thanks! changed that and get a bit higher accuracy from looking at it. Do you have a plan to update how to train custom dataset using .json as annotation file and also the evaluation code? I am using my own dataset, not one of public datasets listed in your repo. It contains very small persons on the beach, so I am considering retraining from scratch. The pretrained model couldn't handle cases that part of human bodies are hidden by sea. Really appreciate your quick reply.

**Comment by hasanirtiza** [comment]
Training on a custom dataset should be similar to training on any of the datasets provided in the repo. Create a .json file or your custom dataset and try with the hyper parameters. {perhaps this can be helpful, [How to use my own datasets](https://github.com/hasanirtiza/Pedestron/blob/master/GETTING_STARTED.md#how-to).
